pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/redigerbar/BekreftelsePaaFlyktningstatus.kt

```diff
@@ -0,0 +1,68 @@
+package no.nav.pensjon.brev.maler.redigerbar
+
+import no.nav.pensjon.brev.api.model.TemplateDescription
+import no.nav.pensjon.brev.api.model.maler.Brevkode
+import no.nav.pensjon.brev.api.model.maler.EmptyRedigerbarBrevdata
+import no.nav.pensjon.brev.maler.fraser.common.Felles
+import no.nav.pensjon.brev.template.Language.Bokmal
+import no.nav.pensjon.brev.template.RedigerbarTemplate
+import no.nav.pensjon.brev.template.dsl.createTemplate
+import no.nav.pensjon.brev.template.dsl.expression.plus
+import no.nav.pensjon.brev.template.dsl.helpers.TemplateModelHelpers
+import no.nav.pensjon.brev.template.dsl.languages
+import no.nav.pensjon.brev.template.dsl.text
+import no.nav.pensjon.brev.template.dsl.textExpr
+import no.nav.pensjon.brevbaker.api.model.BrukerSelectors.foedselsnummer
+import no.nav.pensjon.brevbaker.api.model.FellesSelectors.avsenderEnhet
+import no.nav.pensjon.brevbaker.api.model.FellesSelectors.bruker
+import no.nav.pensjon.brevbaker.api.model.FoedselsnummerSelectors.value
+import no.nav.pensjon.brevbaker.api.model.LetterMetadata
+import no.nav.pensjon.brevbaker.api.model.NAVEnhetSelectors.navn
+
+
+@TemplateModelHelpers
+object BekreftelsePaaFlyktningstatus : RedigerbarTemplate<EmptyRedigerbarBrevdata> {
+
+    override val kode = Brevkode.Redigerbar.PE_BEKREFTELSE_PAA_FLYKTNINGSTATUS
+    override val kategori: TemplateDescription.Brevkategori =
+        TemplateDescription.Brevkategori.INNHENTE_OPPLYSNINGER // TODO
```

Comment:
Om du har TODO her fordi du ikke har bekreftet at det er riktig kategori, så kan du finne den originale brevkoden i `no.nav.pensjon.brev.skribenten.services.BrevmalServiceKt#kategoriOverrides`. I dette tilfellet så ser det du har gått for riktig ut.